client/src/main/scala/org/zalando/nakadi/client/scala/Connection.scala

```diff
@@ -154,7 +154,7 @@ sealed class ConnectionImpl(val host: String,
   }
 
   def executeCall(request: HttpRequest): Future[HttpResponse] = {
-    logger.debug("executingCall {}", request)
+    logger.debug("executingCall {}", request.method)
```

Comment:
Good point, however this method prints the URI together with the Endpoint. Besides that, we are customizing the request with only 2 extra headers Content-type(application/json and application/octet-stream-x) And Authorisation.

The received responses+headers are logged(debug) in the ClientImpl.scala class.